src/GraphQL/Validation/ValidationContext.cs

```diff
@@ -62,7 +63,10 @@ internal void Reset()
         public bool HasErrors => _errors?.Count > 0;
 
         /// <inheritdoc cref="ExecutionOptions.Variables"/>
-        public Inputs? Variables { get; set; }
+        public Inputs Variables { get; set; } = null!;
```

Comment:
I changed to non-null and added `?? Inputs.Empty` in `DocumentExecutor`. The same for `Extensions`.

Reply:
Ok, I guess.